sdk/appplatform/mgmt/pom.xml

```diff
@@ -0,0 +1,139 @@
+<!--
+ Copyright (c) Microsoft Corporation. All rights reserved.
+ Licensed under the MIT License. See License.txt in the project root for
+ license information.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+      <groupId>com.azure.resourcemanager</groupId>
+      <artifactId>azure-resourcemanager-parent</artifactId>
+      <version>2.0.0-SNAPSHOT</version>
+      <relativePath>../../management/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>azure-resourcemanager-appplatform</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Microsoft Azure SDK for App Platform Management</name>
+    <description>This package contains Microsoft Azure App Platform Management SDK.</description>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+    <licenses>
+      <license>
+        <name>The MIT License (MIT)</name>
+        <url>http://opensource.org/licenses/MIT</url>
+        <distribution>repo</distribution>
+      </license>
+    </licenses>
+
+    <scm>
+      <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+      <connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
+      <tag>HEAD</tag>
+    </scm>
+
+    <properties>
+      <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+      <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
+    </properties>
+
+    <developers>
+      <developer>
+        <id>microsoft</id>
+        <name>Microsoft</name>
+      </developer>
+    </developers>
+
+    <dependencies>
+      <dependency>
+        <groupId>com.azure.resourcemanager</groupId>
+        <artifactId>azure-resourcemanager-resources</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+      </dependency>
+      <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-storage-file-share</artifactId>
+        <version>12.5.0</version>
```

Comment:
Should we move the dependency to parent pom so the version control only happen there?

Reply:
No, we'll separate all dependency into package pom later. So I don't add it to parent pom anymore

Reply:
Although, it is not the final shape for the `pom.xml`